src/core/initialize.ts

```diff
@@ -59,6 +59,14 @@ export async function initialize(options: {
   input.template ??= defaultTemplate;
   input.packageManager ??= options.packageManager;
 
+  const isExists = await fs.pathExists(input.directory);
+  if (isExists) {
+    const isEmpty = (await fs.readdir(input.directory)).length === 0;
+    if (!isEmpty) {
+      consola.warn(`The directory ${input.directory} is not empty. Aborted.`);
```

Comment:
Let's use `consola.error` and the absolute path instead.

```suggestion
      consola.error(`The directory ${path.resolve(input.directory)} is not empty. Aborted.`);
```